### ISSUE TITLE
Superfluous response

### ISSUE THREAD

**Issue by KonstantinosNikopoulos** [fstab/grok_exporter#125]
I have this config:

```
global:
  config_version: 3
input:
  type: file
  paths:
  - ./example/Data/maillog/*
  readall: true 
imports:
- type: grok_patterns
  dir: ./patterns
grok_patterns:
  - 'EXIM_MESSAGE [a-zA-Z ]*'
metrics:
   - type: counter
     name: maillog_counter
     help: maillog counter (maillog)
     match: '%{MONTH:month} +%{MONTHDAY:day} %{TIME:time} %{SYSLOGHOST:machine} %{DATA:msg}: (?:%{DATA:msg_id}: message-id=%{GREEDYDATA:message_id}|%{DATA:msg_id}: uid=%{NUMBER:uid} from=%{GREEDYDATA:from}|%{DATA:msg_id}: from=%{DATA:from}, size=%{DATA:size}, %{GREEDYDATA}|%{DATA:msg_id}: to=%{DATA:to}, orig_to=%{DATA:orig_to}, relay=%{DATA:relay}, delay=%{DATA:delay}, delays=%{DATA:delays}, dsn=%{DATA:dsn}, status=%{DATA:status} %{GREEDYDATA}|%{DATA:msg_id}: removed|%{WORD:priority} %{GREEDYDATA:full_msg}|%{WORD:priority}: %{GREEDYDATA:full_msg})'
     path: ./example/Data/maillog/*
     labels:
         status: '{{.status}}'
         from: '{{.from}}'
         to: '{{.to}}'
         relay: '{{.relay}}'
         delay: '{{.delay}}'
         priority: '{{.priority}}'
         unix_timestamp: '{{$a:="0"}}{{$b:=.day}}{{$c:="/"}}{{$d:=.month}}{{$e:="/2020:"}}{{$f:=.time}}{{if eq .day "1" "2" "3" "4" "5" "6" "7" "8" "9"}}{{timestamp "02/Jan/2006:15:04:05" (print $a $b $c $d $e $f)}}{{else}}{{timestamp "02/Jan/2006:15:04:05" (print $b $c $d $e $f)}}{{end}}'
server:
  protocol: http
  port: 9144
```
When i start the server all works fine for a while. All metrics appear in grok and prometheus. But after some minutes metrics don't appear in prometheus and i get this error multiple times in terminal:

```
2020/08/21 09:29:38 http: superfluous response.WriteHeader call from github.com/fstab/grok_exporter/vendor/github.com/prometheus/client_golang/prometheus/promhttp.(*responseWriterDelegator).WriteHeader (delegator.go:58)
2020/08/21 09:29:43 http: superfluous response.WriteHeader call from github.com/fstab/grok_exporter/vendor/github.com/prometheus/client_golang/prometheus/promhttp.(*responseWriterDelegator).WriteHeader (delegator.go:58)
2020/08/21 09:29:48 http: superfluous response.WriteHeader call from github.com/fstab/grok_exporter/vendor/github.com/prometheus/client_golang/prometheus/promhttp.(*responseWriterDelegator).WriteHeader (delegator.go:58)
2020/08/21 09:29:53 http: superfluous response.WriteHeader call from github.com/fstab/grok_exporter/vendor/github.com/prometheus/client_golang/prometheus/promhttp.(*responseWriterDelegator).WriteHeader (delegator.go:58)
```

**Comment by fstab** [comment]
This bug seems to be fixed in `client_golang` v1.7.1, see [https://github.com/prometheus/client_golang/blob/v1.7.1/prometheus/promhttp/delegator.go#L58](https://github.com/prometheus/client_golang/blob/v1.7.1/prometheus/promhttp/delegator.go#L58).

It looks like your version of `grok_exporter` uses an older `client_golang` dependency.

Could you update `grok_exporter` to the current version [v1.0.0.RC4](https://github.com/fstab/grok_exporter/releases)? The current `grok_exporter` release is bundled with `client_golang` v1.7.1 and should fix the issue.

**Comment by KonstantinosNikopoulos** [comment]
Yes you are right, I have 1.0.0.RC3 version
